### PR TITLE
Fix OpenHours import

### PR DESCRIPTION
**Pull request by gnclmorais**
When updating this gem’s version to 1.0 on a project, I kept running into the same issue:

```rb
  cannot load such file -- yelp/fusion/responses/models/openHours
```

After a quick search in the gem’s source code, I’ve noticed some unexpected capitalisation that I was not expecting. Thanks to it, we were basically trying to load the wrong file. Changing it to snake_case fixed the loading issue for my app.

### CODE REVIEW

File: lib/yelp/fusion/responses/models/hours.rb [hunk 1]
```diff
@@ -38,4 +38,4 @@ def initialize(json)
       end
     end
   end
-end
\ No newline at end of file
+end
```
Comment (gnclmorais) [inline lib/yelp/fusion/responses/models/hours.rb:41]: Sorry about this line change, it is automatic through my editor. Feel free to ignore/drop it. 🙇🏻‍♂️

**Comment by erikgrueter1** [comment]
Wow! Thank you!

**Comment by gnclmorais** [comment]
Will this be rolled out into a new gem version soon? It would be great to upgrade our version from a pre-release to 1.1 or something! 🙌🏻

**Comment by erikgrueter1** [comment]
yup! Give me a day Thanks so much for contributing! Question, how are you currently using this gem?

-Erik

**Comment by gnclmorais** [comment]
It’s an older feature we have, but we try to show interesting places near universities: https://app.bridge-u.com/universities/harvard-university

**Pull request merged**